packages/explorer/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@development-framework/explorer",
```

Comment:
could we do a clean up on dependencies here?
I suspect there are many unused ones